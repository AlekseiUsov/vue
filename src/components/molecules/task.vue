<template>
    <li :class="$style.task">
        <checkbox />
        <close />
    </li>
</template>

<script>
import checkbox from '@/components/atoms/checkbox';
import close from '@/components/atoms/close';


export default {
    components: {
        checkbox,
        close,
    }
};
</script>

<style lang="scss" module>
@import "@/assets/main.scss";

.task {
    display: flex;
    justify-content: space-between;
    padding: 0.25rem;
    box-sizing: border-box;
    border-radius: 0.5rem;
    background-color: $light-beige;
    margin-top: 1rem;
}

.task:first-child {
    margin-top: 0;
}
</style>