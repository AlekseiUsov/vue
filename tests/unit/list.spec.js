import { shallowMount } from "@vue/test-utils";
import task from "../../src/components/molecules/ToDoTask.vue";
import list from "../../src/components/organisms/ToDoList.vue";

describe("ToDoList.vue", () => {
    const wrapper = shallowMount(list);
    expect(wrapper).toMatchSnapshot();
})
