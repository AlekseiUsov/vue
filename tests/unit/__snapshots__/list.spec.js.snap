// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToDoList.vue тестируем отрисовку листа 1`] = `
<li><label><input type="checkbox"> <span></span></label>

  <div><span></span></div>
</li>
`;
